qipr_approver/approver/templates/templatetags/form_input.html

```diff
@@ -1,4 +1,4 @@
 <div class="form-input-wrapper input_{{ name }} ">
     <label for="{{ name }}">{{ label }}</label>
-    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" {{ is_disabled }}/>
+    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" placeholder="{{ placeholder }}" {{ is_disabled }}/>
```

Comment:
Regarding the other pull request which I closed, making the username readonly is only solves half of the problem. We are using the data in that field to do a lookup on the server. Even if they can't click/type in it, they can change it and cause the application to error. Probably need to get that information out of the session or something.